fontra2fontir/src/source.rs

```diff
@@ -224,4 +273,38 @@ mod tests {
             glyph_names
         );
     }
+
+    #[test]
+    fn fontdata_of_minimal() {
+        let font_data =
+            FontData::from_file(&testdata_dir().join("minimal.fontra/font-data.json")).unwrap();
+        assert_eq!(1000, font_data._units_per_em);
+        assert_eq!(
+            vec![("Weight", Tag::from_be_bytes(*b"wght"), 200.0, 200.0, 900.0),],
```

Comment:
doesn't `Tag::new(b"wght")` do the same with less characters?

Reply:
oh yes, good call, `new` is the blessed constructor.